react/features/remote-video-menu/components/AbstractKickRemoteParticipantDialog.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+
+import { Component } from 'react';
+
+import {
+    createRemoteVideoMenuButtonEvent,
+    sendAnalytics
+} from '../../analytics';
+import { kickParticipant } from '../../base/participants';
+
+type Props = {
+
+    /**
+     * The Redux dispatch function.
+     */
+    dispatch: Function,
+
+    /**
+     * The ID of the remote participant to be kicked.
+     */
+    participantID: string,
+
+    /**
+     * Function to translate i18n labels.
+     */
+    t: Function
+};
+
+/**
+ * Abstract dialog to confirm a remote participant kick action.
+ */
+export default class AbstractKickRemoteParticipantDialog
```

Comment:
Can we call this `KickParticipantDialog` instead? It's pretty obvious we are dealing with remote participants.

Reply:
Well, it's a matter of consistency vs simplification :)

Consistency: The mute dialog is also named MuteRemoteParticipantDialog, so we can stay consistent with that (or rename that too. Question if that's misleading without the "remote" word or not) or...
Simplification: Rename this dialog, but then see previous point.

What do you think?

Reply:
Let's stay consistent then, good point!